### PR TITLE
FLOE-mobile-responsive: preference panel and images aligned for mobile devices

### LINKED ISSUE #116: Image is not alligned properly in mobile.
Image is not alligned properly when viewed in mobile(android device). Image are shifted or align more towards the right side and not the center.
c2979187-ce2f-4bec-a0ab-808ce88cf66d

### LINKED ISSUE #118: Show preferences panel bar doesn't fit the small screens.
## The show preferences panel doesn't fit with the screen width.
Show preferences panel does not fit the width of the screen on small scale devices(width < 365px).

### To reproduce
1. Go to  https://floeproject.org
2. Open Chrome dev tools and click on the toggle device toolbar icon.
3. Scroll and set the width less than 365px and then see the error.

### Expected behavior
The show preferences panel div should occupy the 100% width of the device screen.

## 1. iPhone 5/SE (Chrome dev tools simulator)
image

## 2.  Samsung galaxy S5 (Chrome dev tools simulator)
image

## Technical details
This bug is evident because the size of the images doesn't scale down when the width of the device goes small.

### PR DESCRIPTION
**Pull request by thisispratt**
Fixes #118 and also fixes #116 
### 1.  #118 
image

### 2.  #116 
image

### CODE REVIEW

**Comment by sachin10101998** [comment]
@thisispratt Do not fix multiple unrelated issues in the same PR. 
This causes two problems:
1) It's tough to track changes in different files for different issues. It's confusing to know what changes were made in order to fix which issue. 
2) Fixes to different issues can be merged separately thereby maintaining order.

**Comment by thisispratt** [comment]
> @thisispratt You should not fix multiple unrelated issues in the same PR.
> This causes two problems:
> 
> 1. It's tough to track changes in different files for different issues. It's confusing to know what changes were made in order to fix which issue.
> 2. Fixes to different issues cannot be merged separately making it tough to maintain order.

@sachin10101998  These issues are related to one another. Issue #118 is a repercussion of #116 
Thus solving them together.

**Comment by jobara** [comment]
@thisispratt could you describe more why https://github.com/fluid-project/floeproject.org/issues/116 causes https://github.com/fluid-project/floeproject.org/issues/118. Also what is the change in the footer for?

**Comment by thisispratt** [comment]
> @thisispratt could you describe more why #116 causes #118. Also, what is the change in the footer for?

Images on the home page were not responsive, thus giving rise to both the issues.

Also, the text which was inline with the  floe-hewlett-logo was flowing out of the container in mobile devices, Hence, the change in the footer.

**Comment by jobara** [comment]
> > @thisispratt could you describe more why #116 causes #118. Also, what is the change in the footer for?
> 
> Images on the home page were not responsive, thus giving rise to both the issues.

How did the images not being responsive lead to this issue?

> 
> Also, the text which was inline with the floe-hewlett-logo was flowing out of the container in mobile devices, Hence, the change in the footer.

Was this related, or should it have been filed under a separate issue?

**Comment by thisispratt** [comment]
> How did the images not being responsive lead to this issue?
> 

Since the images are not responsive so they are flowing out of the container in small scale devices thus extending out to the rightmost side of the viewport first and not letting the container occupy the 100% width of the device.
image



> > Also, the text which was inline with the floe-hewlett-logo was flowing out of the container in mobile devices, Hence, the change in the footer.
> 
> Was this related, or should it have been filed under a separate issue?

Yes, this was also related as the footer text caused the same issue.

**Comment by sachin10101998** [comment]
@thisispratt, @jobara I just properly looked into this issue and I believe that these two issues are not related. You didn't have to make any changes in the footer. This issue should be solved by fixing max-widths of images and containers instead of explicitly changing the size of images.
 Look at #128. The footer issue along with the preference panel bug is solved without making any changes to the footer. 
Moreover, the images are responsive and behave as they should on changing screen sizes.
Screenshots:
1) 
For iphone 5/Se, the prefernce bar:
Screenshot from 2020-03-04 00-04-26

2) 
Responsive images not extending to rightmost side. 
Screenshot from 2020-03-04 00-02-15

3)
Screenshot from 2020-03-04 00-01-54

4)

Screenshot from 2020-03-04 00-01-35

File: css/style.css [hunk 1]
```diff
@@ -562,6 +562,30 @@ figcaption details p {
...
+    .floe-header h1{
+    font-size: 2.8em;
+    }
+
+    .floe-header-tagline{
+        line-height: normal;
+        margin: 3vw 0vw;
+    }
+
+}
```
Comment (sachin10101998) [inline css/style.css]: I changed the contrast from preferences option and this is what the iPhone 5/SE simulator showed.
Screenshots:
Screenshot from 2020-03-04 00-11-23
Screenshot from 2020-03-04 00-11-34

You forgot to make the changes to all available themes.

File: index.html [hunk 2]
```diff
@@ -312,7 +312,11 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
                                 <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>.<br/>
                             </p>
                             <p>
-                                <span class="floe-hewlett-logo">Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.</span>
+                                <span class="floe-hewlett-logo">
+                                    <p>
+                                        Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.
+                                    </p>
+                                </span>
```
Comment (sachin10101998) [inline index.html]: These changes show no effect on the footer for smaller screens. I am unable to understand how adding p tags is related to the overflowing of images in the content part of the website or the preferences column.
1)
Screenshot from 2020-03-04 00-21-30
2)
Screenshot from 2020-03-04 00-21-47

Comment (aashimgarg) [reply index.html]: adding paragraph tags shows no effect. And in fact you only gave indentation to this.

Comment (aashimgarg) [reply index.html]: Adding paragraph tag and adding indentation will show no effect.

Comment (sachin10101998) [reply index.html]: > adding paragraph tags shows no effect. And in fact you only gave indentation to this.

@aashimgarg This is my review of changes proposed by @thisispratt. I didn't propose to add any p tags. 

> Adding paragraph tag and adding indentation will show no effect.

That's exactly what I pointed out in the comment above and this review.

**Comment by jobara** [comment]
@thisispratt, @aashimgarg and @sachin10101998, thanks again for discussing the issues. I think it would be helpful to go back to the issues and describe in detail what what is causing the issues, including references to the code and other resources as necessary. 

Also, remember to always be respectful in your communications. It can be hard to interpret tone and intention in text based communications, so be cautious of how your words may be interpreted and understood.

**Comment by aashimgarg** [comment]
Sorry @jobara . I will makes sure that it will not happen again.

**Pull request closed without merging**